src/validationinterface.cpp

```diff
@@ -6,6 +6,8 @@
 #include "validationinterface.h"
 #include <primitives/certificate.h>
 
+using namespace boost::placeholders;
```

Comment:
Another small stylistic comment here: I believe using this namespace explicitly where needed might make the code  more readable. On the other hand, I see this is currently aligned with zcash

Reply:
Even though using namespace in this way is not a best practice, I would keep this one as an exception just for two reasons:
- keeping the code aligned with Zcash (as you already mentioned)
- not cluttering the file with tens of namespaces for very small placeholder names (_1, _2, _3, etc.)

Reply:
Alright, makes sense.